snapcraft.yaml

```diff
@@ -650,6 +650,30 @@ parts:
       - bin/nvidia-container-cli*
       - lib/libnvidia-container*.so*
 
+  nvidia-container-toolkit:
+    source: https://github.com/NVIDIA/nvidia-container-toolkit
+    source-depth: 1
+    source-tag: v1.15.0
+    source-type: git
+    plugin: make
+    override-prime: |-
+      [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ] && exit 0
+      craftctl default
+    override-pull: |-
+      [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ] && exit 0
+      craftctl default
+    override-build: |-
+      [ "$(uname -m)" != "x86_64" ] && [ "$(uname -m)" != "aarch64" ] && exit 0
+      set -ex
+
+      make binaries
+      mkdir -p "${CRAFT_PART_INSTALL}/bin/"
+      cp nvidia-ctk "${CRAFT_PART_INSTALL}/bin/"
+    organize:
+      usr/bin/: bin/
+    prime:
+      - bin/nvidia-ctk
```

Comment:
Is this a static binary (as in 0 `.so` files showing in `ldd`'s output)?

Reply:
no it's not:
```
$ ldd nvidia-ctk 
        linux-vdso.so.1 (0x00007ffcaa0d0000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x000072888c400000)
        /lib64/ld-linux-x86-64.so.2 (0x000072888c6c5000)
```

Reply:
Ah thanks for checking. I think those are special `.so` you don't need to prime so I think this is all good then.

Reply:
fixed